src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,57 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const clickHandler = (tab: Tab) => {
+    if (selectedTabId !== tab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  const selectedTab = tabs.find((tab) => (tab.id === selectedTabId)) || tabs[0];
```

Comment:
useMemo

Reply:
@ihor-jpeg, As I understand, `useMemo` should be used for a variable which is passed as a props to another component (so that the child component does not re-render each time a parent component is rendered and a new copy of the variable is created).

In the current scenario, the `selectedTab` is not being passed to any other component, so does it still add any value to use `useMemo`

Reply:
Done.